esp_flasher/own_esptool.py

```
@@ -3358,6 +3359,235 @@ def hard_reset(self):
+    MEMORY_MAP = [
+        [0x00000000, 0x00010000, "PADDING"],
+        [0x40000000, 0x54000000, "DROM"],
+        [0x2F000000, 0x2F080000, "DRAM"],
+        [0x2F000000, 0x2F080000, "BYTE_ACCESSIBLE"],
+        [0x2F800000, 0x2F850000, "DROM_MASK"],
+        [0x2F800000, 0x2F850000, "IROM_MASK"],
+        [0x40000000, 0x54000000, "IROM"],
+        [0x2F000000, 0x2F080000, "IRAM"],
+        [0x2E000000, 0x2E008000, "RTC_IRAM"],
+        [0x2E000000, 0x2E008000, "RTC_DRAM"],
+    ]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)